src/components/cl/basic/cl_basic_coll.c

```diff
@@ -11,18 +11,14 @@ ucc_status_t ucc_cl_basic_coll_init(ucc_base_coll_args_t *coll_args,
                                     ucc_base_team_t *team,
                                     ucc_coll_task_t **task)
 {
-    ucc_cl_basic_team_t    *cl_team = ucc_derived_of(team, ucc_cl_basic_team_t);
-    ucc_base_coll_init_fn_t init;
-    ucc_base_team_t        *bteam;
-    ucc_status_t            status;
-    status =
-        ucc_coll_score_map_lookup(cl_team->score_map, coll_args, &init, &bteam);
+    ucc_cl_basic_team_t *cl_team = ucc_derived_of(team, ucc_cl_basic_team_t);
+    ucc_status_t         status;
+
+    status = ucc_coll_score_map_init(cl_team->score_map, coll_args, task);
```

Comment:
given than  we are initializing the coll->init() inside and score_map is an asrugment for initilaization, may be we can rename it to ucc_coll_init()/ WDYT?

Reply:
sounds good